api_review/ColorPickerDisplayMode/ColorPickerDisplayModeSpec.md

```diff
@@ -0,0 +1,118 @@
+The aim of this new API is, to make the ColorPicker more versatile to use and provide different layouting ways to maximize the space available.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+With the DisplayMode API for the ColorPicker, the ColorPicker is able to either layout itsself horizontally or vertically. 
+In addition to this the ColorPicker can also choose its layout based on the space available.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+A standard ColorPicker using DisplayMode "Vertical":
+```xaml
+<muxc:ColorPicker IsAlphaEnabled="True" DisplayMode="Vertical"/>
+```
+![Vertical ColorPicker](./ColorPicker_VerticalMode.png)
+
+Standard ColorPicker using DisplayMode "Horizontal":
+```xaml
+<muxc:ColorPicker IsAlphaEnabled="True" DisplayMode="Horizontal"/>
+```
+![Horizontal ColorPicker](./ColorPicker_HorizontalMode.png)
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+When using `DisplayMode="Auto"` the ColorPicker will prefer height over width, meaning if has enough vertical space it will choose a vertical layout, regardless of the horizontal space available.
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+DisplayMode: The DisplayMode determining wether to display horizontally or vertically.
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+```c#
+
+[webhosthidden]
+unsealed runtimeclass ColorPicker
+{
+    /// Gets or sets the DisplayMode of the ColorPicker
+    ColorPickerDisplayMode DisplayMode = ColorPickerDisplayMode.Vertical;
```

Comment:
As I suggested in the comments, perhaps

```suggestion
    ColorPickerOrientation Orientation = ColorPickerOrientation.Vertical;
```

Reply:
As you suggested, I updated the proposal to use the existing Orientation enum. I also updated the property name to "Orientation".